src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs

```diff
@@ -281,7 +281,7 @@ internal static Diagnostic Create(DiagnosticInfo info)
         /// <summary>
         /// Returns true if the diagnostic has a source suppression, i.e. an attribute or a pragma suppression.
         /// </summary>
-        public abstract bool IsSuppressed { get; }
+        public virtual bool IsSuppressed { get { throw new NotImplementedException(); } }
```

Comment:
@jaredpar That might be true. I went through all new abstract methods on the public API and applied the same fix everywhere.